components/arm.py

```diff
@@ -177,6 +178,13 @@ def __init__(self) -> None:
             4,
             wpilib.Color8Bit(68, 117, 26),
         )
+
+        # Hall effector
+        self.hall_effector1 = DigitalInput(SparkMaxIds.hall_effector1)
+        self.hall_effector2 = DigitalInput(SparkMaxIds.hall_effector2)
```

Comment:
I thought we were going to wire these hall effect sensors as limit switches to the motor controller?